packages/ui/Routable/Link.js

```diff
@@ -36,6 +95,42 @@ const LinkBase = kind({
 	}
 });
 
+/**
+ * A higher-order component that adds support to a component to handle navigates actions.
+ * When using Linked component you must give the `path` property. And the component should forward `onClick` event.
+ *
+ * ```
+ * const CustomItemBase = kind({
+ *	  name: 'CustomItem',
+ *
+ *	  handlers: {
+ *		  onClick: handle(
+ *			  forward('onClick')
+ *		  )
+ *	  },
+ *
+ *	  render: ({...rest}) => {
+ *		  return (
+ *			  <div {...rest} />
+ *		 );
+ *	  }
+ * });
+ *
+ * const CustomItem = Linkable(CustomItemBase);
+ *
+ * const Main = () => (
+ *	  <div>
+ *		  <CustomItem path="./about">About</CustomItem>
+ *		  <CustomItem path="./faq">FAQ</CustomItem>
+ *	  </div>
+ * );
+ * ```
+ *
+ * @class Linkable
+ * @memberof ui/Routable
+ * @hoc
+ * @public
+ */
 const Linkable = hoc({navigate: 'onClick'}, (config, Wrapped) => {
```

Comment:
It has `defaultConfig` {navigate: 'onClick'}. Please add description for this.

Reply:
It's added. 👌